### DISCUSSION TITLE
Shortcut Zoom + PLUS did not work

### DISCUSSION THREAD

**Discussion by Joonath** [AppGround-io/bluetooth-keyboard-and-mouse-support#213] [Bugs & Problems]
Really love the app so far! With all of functionalities provided, it just feels (almost) like trackpad all over again.

Today I stumbled upon a "broken" custom shortcut. I was trying to make a shortcut for zooming purposes with the combination of both `Ctrl + PLUS` and `Ctrl + MINUS`.

The zoom out (Ctrl + MINUS) is working, however it's a different case for zoom in. So far I've tried the combination on browsers (Chrome, Firefox), I have not yet tried outside browser category.

Looking forward for the fixes!

===
A bit detail of my devices, if it helps:
*Smartphone, app version*
- Android 10, HUAWEI P40 5G ANA NX9
- App version 4.7.0, Premium through Google Play Store in-app purchase

*Laptop spec*
- ASUS VivoBook Win 10 OEM Home Edition
- AMD Ryzen 5 2500U
- 1TB HDD/8 GB RAM

**Comment by Staini** [comment]
I'm using the same firefox shortcut and it's working.
I've noticed it's important to use 'Ctrl + PLUS' and not 'PLUS + Ctrl'. Maybe this is the problem?

**Comment by Joonath** [comment]
Sorry for the late response @Staini . I already did both combination and still wouldn't work. 

But somehow using combination of Ctrl + Numpad+ works. I guess it's the workaround as of now.
